src/views/ArticleBody.jsx

```diff
@@ -48,19 +60,102 @@ function ArticleBody(props) {
           40k claps
         </span>
         <div className="share-grp">
-          <a href="you" className="text-danger"><i className="fas fa-envelope" /></a>
-          <a href="you" className="text-primary"><i className="fab fa-facebook-square" /></a>
-          <a href="you" className="text-white"><i className="fab fa-twitter-square text-info" /></a>
-          <a href="you" className="text-muted"><i className="fas fa-bookmark" /></a>
-          <a href="you" className="text-muted"><i className="fas fa-ellipsis-h" /></a>
+          <div className="share__group__child">
```

Comment:
#### Nitpick
- Let's be consistent with naming our style classes.

Reply:
This has been resolved.